eemployee_storage/employee-storage.sh

```
@@ -0,0 +1,43 @@
+# Elastic object storage and VPC set up for Employee Data Storage
+
+# Set AWS profile to use LocalStack
+
+export AWS_PROFILE=localstack
+
+# Variables configuration
+
+REGION="us-east-1"
+VPC_NAME="employee-data-vpc"
+BUCKET_NAME="employee-data-storage"
+SUBNET_NAME="employee-data-subnet"
```

Shellcheck (0.10.0)

[warning] SUBNET_NAME appears unused. Verify use (or export if used externally).

(SC2034)